bluesky_queueserver/manager/worker.py

```diff
@@ -404,8 +491,6 @@ def init_namespace():
         bec = BestEffortCallback()
         self._RE.subscribe(bec)
 
-        # db = Broker.named('temp')
```

Comment:
Where do we define a databroker instance and subscription now?

Reply:
DB is instantiated in Line 27.